lib/game.rb

```diff
@@ -0,0 +1,240 @@
+require './lib/card'
+require './lib/deck'
+require './lib/player'
+require './lib/turn'
+
+
+
+class Game
```

Comment:
All the extra line breaks make this class really difficult to read through.

Reply:
I'm definitely going to need to work on my getting claustrophobic when things are bunched together more and avoid spacing things out!